docs/ref/models/fields.txt

```diff
@@ -457,6 +457,8 @@ The primary key field is read-only. If you change the value of the primary
 key on an existing object and then save it, a new object will be created
 alongside the old one.
 
+If you specified the primary_key to an actual data field, after :ref:`topics-db-queries-delete` the primary_key will be set to None.
```

Comment:
Maybe:
```
The primary key field is set to ``None`` when
:meth:`deleting <django.db.models.Model.delete>` an object.
```

Reply:
Thanks, I will redo it. I am basically new to contributions, so after I remodel it in my local repo, I commit it and then I need to make a new pull request, right?